### DISCUSSION TITLE
What should mods be able to do?

### DISCUSSION THREAD

**Discussion by lxgr-linux** [lxgr-linux/pokete#67] [Ideas]
## What should mods be able to do?

At some point I was starting to think about adding a mod-loader, and therefore was thinking about what mods should actually be able to/add/modify in the game, and to what extend.

So, what do you think?

**Comment by MaFeLP** [comment]
- Add/Modify:
  - Poketes
  - Attacks
  - Maps
  - Trainers
  - Items
  - Menus
 - Being able to trade these new poketes with other playes that have the required mods installed

**Reply by lxgr-linux** [reply]
First of all, thanks for your reply.

See #68 for the first implementation of this.
~~The addition to trading will be made later.~~
